packages/middleware/README.md

```diff
@@ -0,0 +1,30 @@
+<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/middleware
+
+**A core Josh utility to create middleware**
+
+[![GitHub](https://img.shields.io/github/license/josh-development/utilities)](https://github.com/josh-development/utilities/blob/main/LICENSE)
+[![npm](https://img.shields.io/npm/v/@joshdb/middleware?color=crimson&logo=npm&style=flat-square&label=@joshdb/middleware)](https://www.npmjs.com/package/@joshdb/middleware)
+
+</div>
+
+## Description
+
+A core Josh utility to create middleware.
+
+## Features
+
+- Written in TypeScript
+- Offers CommonJS, ESM, and UMD bundles
+- Fully tested
```

Comment:
Can we discuss this features section, its repetitive in every repo and doesn't contain every cool feature

Reply:
Yeah, basically just placeholders, we could probably do this better later on when we start working on website